code/modules/games/52card.dm

```diff
@@ -6,45 +6,83 @@
 /obj/item/deck/cards
 	name = "deck of cards"
 	desc = "A simple deck of playing cards."
-	icon_state = "deck"
+	icon_state = "deck_nanotrasen_full"
+	card_style = "nanotrasen"
 
-/obj/item/deck/cards/New()
-	..()
-	for(var/suit in list("spades","clubs","diamonds","hearts"))
+/obj/item/deck/cards/build_deck()
```

Comment:
this proc needs to call update_icon at the end. else black cards start off blue until you draw a card.